README.md

```diff
@@ -282,12 +282,63 @@ and `A` components which make use of `src` and `href` props).
 
 One other tip... This totally works:
 
-```javascript
+```jsx
 <glamorous.Div color="blue">
   JSX is pretty wild!
 </glamorous.Div>
 ```
 
+### Theming
```

Comment:
Great docs! Once this is merged, Could you add a link to a CodeSandbox so people can play around with it? Could be based on [the help one](http://kcd.im/glamorous-help)